src/meenu/assignment_15/BankDetails.java

```diff
@@ -0,0 +1,182 @@
+package meenu.assignment_15;
+
+/*A person has total 4 accounts in ICICI, HDFC, BOB, SBI one in each.
+          a) Person can debit or credit amount from respective bank.
+              Hint : Switch case is expected here. [Please watch today's recording again, in case of confusion.]
+
+          b) A Method to show total balance [ICICI balance + HDFC balance + BOB balance + SBI Balance]
+
+          c) A Method to show total credit, debit operations 
+                 output : Total credit operation - 7 times
+                              Total debit operation - 5 times
+
+         d) A Method to show individual bank credit operations.
+                output : ICICI bank Credit Operation - 2
+                         HDFC bank Credit Operation - 2
+		                 SBI bank Credit Operation - 1
+                         BOB bank Credit Operation - 4
+
+         e) A Method to show individual bank debit operations.
+                output : ICICI bank Debit Operation - 2
+                         HDFC bank Debit Operation - 2
+		                 SBI
+*/
+
+import java.util.Scanner;
+
+public class BankDetails {
+	int iciciBalance = 1000;
+	int hdfcBalance = 2000;
+	int bobBalance = 3000;
+	int sbiBalance = 4000;
+	int totalBalance;
+	int totalCreditCount;
+	int totalDebitCount;
+	int iciciCreditCount;
+	int hdfcCreditCount;
+	int bobCreditCount;
+	int sbiCreditCount;
+	int iciciDebitCount;
+	int hdfcDebitCount;
+	int bobDebitCount;
+	int sbiDebitCount;
+
+	void getTotalBalance(int iciciBal, int hdfcBal, int bobBal, int sbiBal) {
+		iciciBalance = iciciBal;
+		hdfcBalance = hdfcBal;
+		bobBalance = bobBal;
+		sbiBalance = sbiBal;
+
+		totalBalance = iciciBalance + hdfcBalance + bobBalance + sbiBalance;
+		System.out.println("Total balance in all account is : " + totalBalance);
+
+	}
+
+	void totalOperation(String bank, String operation, int amount) {
+		switch (operation) {
+		case "debit":
+			switch (bank) {
+			case "icici":
```

Comment:
instead of writing same code block for multiple times, you can write debit() method which accept the two arguments and you can simply call that method.